docker/autograph/autograph_localdev_config.yaml

```diff
@@ -251,3 +251,7 @@ authorizations:
       key: 9vh6bhlc10y63ow2k4zke7k0c3l9hpr8mo96p92jmbfqngs9e7d
       signers:
           - webextensions-rsa-with-recommendation
+
+heartbeat:
+    hsmchecktimeout: 100ms
```

Comment:
We're not using an HSM but I suppose it doesn't hurt to have it...

Reply:
Yea I thought the same thing, I just copied both over from the example config. Fwiw, we are not using a database either, so the db timeout shouldn't apply as well, but I guess that could be an issue to be filed in autograph itself.